extensions/adobe/experience/adcloud/partnerdata.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/partnerdata",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adcloud partner data",
+  "type": "object",
+  "description": "Details about AdCloud partners.",
+  "definitions": {
+    "partnerdata": {
+      "properties": {
+        "xdm:id": {
+          "title": "Search/Marketing partner ID",
+          "type": "int",
+          "description":
+            "Internal mapping of search/marketing platforms/partners to IDs for Adcloud Search."
+        },
+        "xdm:data": {
```

Comment:
Is this some sort of packed/complex multi value data value?

Reply:
Yes, We are planning to use it to store data such as conversion id/ conversion label  that adCloud Search partner such as Google or Facebook will return. The conversion id is a unique identifier that is linked to a segment id. Also, we can use the same segment to create audiences on multiple partner platforms, hence this data-structure